draft-ietf-gnap-core-protocol.md

```diff
@@ -5365,3 +5365,693 @@ justification for the data type representing the same basic kind of thing it ext
 For example, an extension declaring an "array" representation for a field would need
 to explain how the array represents something akin to the non-array element that it
 is replacing. 
+
+# JSON Schema
```

Comment:
General comment: I suggest to **not** maintain this schema in the near future, because it would make changes to the protocol much more painful. Let's revisit it when the protocol settles.

Reply:
I think it should generally stay in sync, but I'd be comfortable with it lagging a bit during times of major churn.